.travis.yml

```diff
@@ -35,74 +35,83 @@ before_install: >
     echo "0" | sudo tee /proc/sys/net/ipv6/conf/all/disable_ipv6;
     echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}'| sudo tee /etc/docker/daemon.json; 
     sudo service docker restart; 
 
     if [[ "${PLATFORM}" == "debian" ]]; then
-      docker build \
+      echo -e " \033[1;33mRebuilding Debian Docker Images";
+      docker build --quiet \
         --build-arg TOKEN=${TOKEN} \
         --build-arg REV=${TRAVIS_COMMIT} \
         --build-arg STAGE=deps -t ${IMAGE_DEPS_DEBIAN} \
         -f $(pwd)/docker/Dockerfile-debian $(pwd)/docker/ ;
     elif [[ "${PLATFORM}" == "centos" ]]; then
-      docker build \
+      echo -e " \033[1;33mRebuilding CentOs Docker Images";
+      docker build --quiet \
         --build-arg TOKEN=${TOKEN} \
         --build-arg REV=${TRAVIS_COMMIT} \
         --build-arg STAGE=deps -t ${IMAGE_DEPS_CENTOS} \
         -f $(pwd)/docker/Dockerfile-centos $(pwd)/docker/ ;
     else
+      echo -e " \033[1;33mPulling existing Docker Images";
       docker pull ${IMAGE_DEPS_DEBIAN}; 
       docker pull ${IMAGE_DEPS_CENTOS};
     fi
   fi
-install: true
-before_script: true
+
 script: >
   if [[ "${TEST_STATUS}" && "${STAGE}" == "compliance" ]]; then
+    echo -e " \033[1;33mRunning Compliance Tests";
     $(pwd)/docker/build.sh -s compliance -t -p $(pwd);
   elif [[ "${PUSH_STATUS}" && "${STAGE_TYPE}" == "release" && "${PLATFORM}" == "debian" ]]; then
-    docker build \
+    echo -e " \033[1;33mBuilding Debian Docker image for Release";
+    docker build --quiet \
       --build-arg TOKEN=${TOKEN} \
       --build-arg REV=${TRAVIS_COMMIT} \
       --build-arg STAGE=release -t ${IMAGE_RELEASE_DEBIAN} -f $(pwd)/docker/Dockerfile-debian $(pwd)/docker/ ;
   elif [[ "${PUSH_STATUS}" && "${STAGE_TYPE}" == "release" && "${PLATFORM}" == "centos" ]]; then
-    docker build \
+    echo -e " \033[1;33mBuilding CentOs Docker image for Release";
+    docker build --quiet \
       --build-arg TOKEN=${TOKEN} \
       --build-arg REV=${TRAVIS_COMMIT} \
       --build-arg STAGE=release -t ${IMAGE_RELEASE_CENTOS} -f $(pwd)/docker/Dockerfile-centos $(pwd)/docker/ ;
   elif [[ "${TEST_STATUS}" && "${PLATFORM}" == "debian" && "${STAGE}" != "compliance" ]]; then
+    echo -e " \033[1;33mRunning Tests against Debian Image";
     docker run \
       -t --rm -v $(pwd):${HOME} --entrypoint bash \
       -e ORIONLD_CACHED_CONTEXT_DIRECTORY=${HOME}/ldcontexts \
       -e CB_SKIP_FUNC_TESTS="${CB_SKIP_FUNC_TESTS}" \
       ${IMAGE_DEPS_DEBIAN} build.sh -s ${STAGE_TYPE} -dqt;
   elif [[ "${TEST_STATUS}" && "${PLATFORM}" == "centos" && "${STAGE}" != "compliance" ]]; then
+    echo -e " \033[1;33mRunning Tests against CentOs Image";
     docker run \
       -t --rm -v $(pwd):${HOME} --entrypoint bash \
       -e ORIONLD_CACHED_CONTEXT_DIRECTORY=${HOME}/ldcontexts \
       -e CB_SKIP_FUNC_TESTS="${CB_SKIP_FUNC_TESTS}" \
       ${IMAGE_DEPS_CENTOS} build.sh -s ${STAGE_TYPE} --os centos -dqt;
   fi
```

Comment:
Any else statement here ?
Perhaps to set an error message or to exit, as nothing has been done and probably something is wrong?

Also, what if the "docker run" fails?

There's a "docker run" in each of the if/elif stamements.

If it fails, we don't know right now.

Perhaps simply add a error=$? after each "docker run" and after the if/elif/then  if error != 0 perform an exit?
[ "error" would be set to some != 0 value in the else branch ]